README.md

```diff
@@ -237,20 +237,16 @@ _MDAs - IMAP/POP3 software_
 * [Dovecot](http://www.dovecot.org/) - IMAP and POP3 server written primarily with security in mind. ([Source Code](https://github.com/dovecot/core)) `MIT/LGPLv2` `C`
 * [Piler](http://www.mailpiler.org/wiki/start) - feature rich open source email archiving solution. ([Source Code](https://bitbucket.org/jsuto/piler)) `GPLv3` `C`
 
-#### Mailing lists
+#### Mailing lists and Newsletters
 
-_Mailing lists servers._
+_Mailing lists servers and mass mailing software - one message to many recipients._
 
+  * [Dada Mail](http://dadamailproject.com/) - A web-based list management system that can be used for announcement lists and/or discussion lists. (Also has a "Pro" version which allows for more/larger mailing lists and branding removal.) ([Source Code](https://github.com/justingit/dada-mail)) `GPLv2` `Perl`
```

Comment:
You need the period outside the bracket otherwise our scanner will flag it as invalid.